Pyjion/pycomp.cpp

```diff
@@ -81,7 +81,7 @@ void CeeInit() {
     CoreClrCallbacks cccallbacks;
     cccallbacks.m_hmodCoreCLR = (HINSTANCE)GetModuleHandleW(NULL);
     cccallbacks.m_pfnIEE = IEE;
-    cccallbacks.m_pfnGetCORSystemDirectory = GetCORSystemDirectoryInternal;
+    //cccallbacks.m_pfnGetCORSystemDirectory = GetCORSystemDirectoryInternal;
```

Comment:
Instead of commenting this out can you update GetCORSystemDirectoryInternal to take a SString& ?  I realize we don't expect this to be called right now, but it'd be nice if it were obvious if it started getting called in the future.